### PR TITLE
Add sync/async responses for remote control status

### PR DESCRIPTION
**Pull request by sphlabs**
* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)

New feature to add response codes for remote control status. Addresses #15 

* **What is the current behavior?** (You can also link to an open issue here)

Status can be set, but querying status does not pass back the response code, and there is no async support when notify: remote is enabled.

* **What is the new behavior (if this is a feature change)?**

Adds a getter and a setter to the remote command. Adds an async handler.

* **Other information**:

### CODE REVIEW

File: src/commands/remote.ts [hunk 1]
```diff
@@ -1,5 +1,33 @@
...
+import { ResponseMessage, NamedMessage } from '../message'
+import { parseBool } from '../util'
+import { AbstractCommand, AbstractCommandNoResponse } from './abstractCommand'
+
+export interface RemoteInfoCommandResponse {
+	enabled?: boolean
+	override?: boolean
+}
+
+//TODO A response is optional, may be a 200. How do we handle this?
```
Comment (mint-dewit) [inline src/commands/remote.ts]: This todo seems unnecessary, since get and set are separated commands now.

**Review by mint-dewit — CHANGES_REQUESTED** [review]

File: src/commands/remote.ts (hunk 1 above)
Comment (sphlabs) [reply src/commands/remote.ts]: Indeed it is, and has been removed.

**Comment by sphlabs** [comment]
Not sure why the dependencies test is failing? Error log doesn't give a lot of clues and I've the PR doesn't change the dependancies.

**Comment by mint-dewit** [comment]
It's just a dev dependency that needs to be updated, something for the maintainers to look at (me). I'll try to release this immediately.

**Pull request merged**